### ISSUE TITLE
Full example?

### ISSUE THREAD

**Issue by stuaxo** [supercodepoet/django-merlin#19]
Would be really good if there was a full example, I.E. a simple django app, with:

Simple 3 step wizard, outcome of 1st step changes subsequent steps.

**Comment by ghost** [comment]
I agree. It is difficult to figure out how to use this for an intermediate django user such as myself. I posted a question on stackoverflow, and implementing the merlin formwizard has proven daunting for more experienced programmers there as well.

My question is located at:   http://stackoverflow.com/questions/14693811/django-formwizard-as-view-method-attributeerror   if anyone has any ideas.  Thanks!
